samples/grpc/shopping-analytics-service-scala/build.sbt

```diff
@@ -20,15 +20,17 @@ Test / parallelExecution := false
 Test / testOptions += Tests.Argument("-oDF")
 Test / logBuffered := false
 
-run / fork := false
+run / fork := true
```

Comment:
um, I also noticed that the newer sbt version didn't work. It started up fine but then just shutdown the ActorSystem immediately.

Reply:
It's the same issue we have heard about since sbt 1.6 or something came out, needs to fork because it doesn't understand the Akka threads should keep it alive, we changed that in quickstarts etc, but maybe we forgot to update the platform samples with the same change (should we backport?)

Forking also means it is a bit trickier with passing the sys prop to choose config, I think I forgot that in this build.sbt but updated the other one, I'll copy that here as well.

Reply:
I works with `sbt.version=1.6.2`. I have created issue https://github.com/akka/akka-platform-guide/issues/799 for the guide.

Reply:
I found the change we did in the quickstarts, but I can't find where we discussed it. Here's the root sbt issue though: https://github.com/sbt/sbt/issues/6767